scripts/cleanup-test-env.sh

```diff
@@ -16,10 +16,33 @@
 
 set -Eeou pipefail
 
+delete_endpoint() {
+    provider=$1
+    set +e
+    count=$(atlas privateEndpoints "${provider}" list --projectId "${clean_project_id}" -o=go-template="{{len .}}")
+    retVal=$?
+    if [ $retVal -ne 0 ]; then
+        count=0
+    fi
+    if [ "${count}" != "0" ]; then
+        echo "Project ${clean_project_id} contains ${provider} endpoints, will start deleting it now and will try to delete the project in the next execution"
+        id=$(atlas privateEndpoints "${provider}" list --projectId "${clean_project_id}" -o=go-template="{{(index . 0).Id}}")
+        atlas privateEndpoints "${provider}" delete "${id}" --force --projectId "${clean_project_id}" || \
+        echo "Failed to delete ${provider}  private endpoint with project ID ${clean_project_id}, endpoint ID: ${id}"
+    fi
+    set -e
+}
+
 projectToSkip="${PROJECT_TO_NOT_DELETE:-NONE}"
 
+if [ -z "${MONGODB_ATLAS_ORG_ID}" ]; then
+  echo "Error: MONGODB_ATLAS_ORG_ID environment variable must be set"
+  exit 1
+fi
+org_id="${MONGODB_ATLAS_ORG_ID}"
+
 # Get all project Ids inside the organization
-projects=$(atlas project ls -o json)
+projects=$(atlas project ls --limit 500 --orgId ${org_id}  -o json)
```

Comment:
nit: According to our documentation https://www.mongodb.com/docs/atlas/reference/atlas-limits/, we can only have max 250 projects per org